linking to egress rules (#2215)
18F/cg-site: _docs/services/relational-database.md

test-oracle          aws-rds   medium-oracle-se2                create succeeded
```

> By default, when new spaces are created in your organization an application security group (ASG) is applied that restricts access to only the internal cloud.gov network. You will need to [update egress traffic](/docs/management/space-egress/) rules to allow for your app to reach the database. 

Reviewer: ```suggestion
By default, when new spaces are created in your organization, an application security group (ASG) is applied that doesn't allow any outgoing traffic. You will need to [update egress traffic]({{ site.baseurl }}{% link _docs/management/space-egress.md %}) rules to allow for your app to reach the database. 
```

@LindsayYoung We updated our settings so that the default ASG actually allows no egress. You could be seeing different behavior in your org if it pre-dates when we made this change.

Author: Thanks for the updated text! Sorry I did not see your comment earlier.
